openstudiocore/src/utilities/core/UUID.i

```diff
@@ -12,16 +12,13 @@
 namespace openstudio{
 
   #ifdef SWIGRUBY
-    %rename("nil?") isNull();
+    %rename("nil?") is_nil();
```

Comment:
I was originally confused thinking:

    if !model_object.handle

would work, but I forgot that Ruby doesn't implicitly call `.nil?` in that case.  As for the name, I've never called `.nil?`, I always use `isNull`, so we should keep all of those.  What is the benefit of also adding `is_nil`?  Is there any reason to add a third method for that?